src/sentry/api/endpoints/integrations/sentry_apps/internal_app_token/details.py

```diff
@@ -70,3 +71,28 @@ def delete(self, request: Request, sentry_app, api_token) -> Response:
         )
 
         return Response(status=204)
+
+
+# TODO: when the above endpoint is deprecated, simply move the convert method.
+# This endpoint handler utilizes the id instead.
+@control_silo_endpoint
+class NewSentryInternalAppTokenDetailsEndpoint(SentryInternalAppTokenDetailsEndpoint):
+    owner = ApiOwner.INTEGRATIONS
+    publish_status = {
+        "DELETE": ApiPublishStatus.EXPERIMENTAL,
```

Comment:
This is simply to help with the migration efforts of the endpoint. This "endpoint" and class will be removed once the FE has fully transitioned over to delete by id.